backend/routes/connectionRoutes.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const {
+  sendRequest,
+  allRequests,
+  accept,
+  reject,
+} = require("../controllers/connectController");
+const router = express.Router();
+
+// Send a connection request endpoint
+router.post("/send-request", sendRequest);
+
+// View all requests for a user endpoint
+router.get("/requests/:userId", allRequests);
```

Comment:
these are routes should be protected by middleware no? In current scenario, anyone knowing userId will be able to fetch/accept/reject any other's request

Reply:
Ok i will update.